### DISCUSSION TITLE
如何调整倾斜摄影的高度？

### DISCUSSION THREAD

**Discussion by 987763485** [zouyaoji/vue-cesium#472] [Q&A] [answered]
我使用supermap的rest发布的倾斜摄影，在vue-cesium中使用，无法调整高度

**Comment by zouyaoji** [comment]
vue-cesium引的超图的包 然后应该加的 s3m ？

**Reply by 987763485** [reply]
我是按照官网提供的示例直接替换url的
![image](https://github.com/zouyaoji/vue-cesium/assets/22728641/fa943f3a-4f01-4412-a3da-7d41bc5553f8)

**Reply by songxiang** [reply]
首先获取到倾斜摄影图层（S3MTilesLayer）,然后通过设置style3D的底部高程来调整高度，参考代码；http://support.supermap.com.cn:8090/webgl/Cesium/docs/Documentation/S3MTilesLayer.html?classFilter=S3MTilesLayer
![1700206440757](https://github.com/zouyaoji/vue-cesium/assets/22089247/99ada222-8d72-4952-8f40-1ecbc1ebb7ea)

**Reply by 987763485** [reply, marked as answer]
感谢指导
我通过你们提供的信息设置成功了
![image](https://github.com/zouyaoji/vue-cesium/assets/22728641/8be9a9d0-28c8-4172-9a4a-cdf45d4d2e84)
